packages/react-composites/playwright.config.common.ts

```diff
@@ -60,21 +60,21 @@ const config: PlaywrightTestConfig = {
 
   projects: [
     {
-      name: 'Desktop Chrome',
+      name: 'desktop-chrome',
```

Comment:
I gave up trying to shell escape space properly on bash, cmd prompt, power shell...

Just get rid of these spaces. :/

Reply:
Reverted this change. Moved to own PR: https://github.com/Azure/communication-ui-library/pull/2162 because it causes all snapshot files to be renamed, making it impossible to review the actual snapshot diffs in this PR.